src/modules/account/accountState.js

```diff
@@ -483,6 +483,23 @@ class AccountState {
     return newWalletInfo.id
   }
 
+  listSplittableWalletTypes (walletId) {
+    const walletInfo = this.allKeys.find(
+      walletInfo => walletInfo.id === walletId
+    )
+    if (!walletInfo) throw new Error(`Invalid wallet id ${walletInfo.id}`)
+    const plugin = getCurrencyPlugin(
+      this.ai.props.output.currency.plugins,
+      walletInfo.type
+    )
```

Comment:
There is no error handling for if `plugin` is `undefined`.

Reply:
added